src/app/payment/paymentMethod/CheckoutcomCustomFormFields.tsx

```diff
@@ -44,21 +52,39 @@ const Sepa: FunctionComponent<CheckoutcomAPMFormProps> = ({method}) => {
     }
 
     return (<>
+        <div className="checkoutcom-sepa-column-container">
+            <div className="checkoutcom-sepa-column-content">
+                <h4 className="checkoutcom-sepa-title">
+                    <TranslatedString id="payment.checkoutcom_sepa_creditor_title" />
+                </h4>
+                <h5 className="checkoutcom-sepa-title">{ creditor.sepaCreditorCompanyName }</h5>
+                <p className="checkoutcom-sepa-line">{ creditor.sepaCreditorAddress }</p>
+                <p className="checkoutcom-sepa-line">{ `${creditor.sepaCreditorPostalCode} ${creditor.sepaCreditorCity}` }</p>
+                <p className="checkoutcom-sepa-line">{ creditor.sepaCreditorCountry }</p>
+                <br />
+                <p className="checkoutcom-sepa-line">
+                    <TranslatedString data={ {creditorId: creditor.sepaCreditorIdentifier} } id="payment.checkoutcom_sepa_creditor_id" />
+                </p>
+            </div>
+            <div className="checkoutcom-sepa-column-content">
+                <h4 className="checkoutcom-sepa-title">
+                    <TranslatedString id="payment.checkoutcom_sepa_debtor_title" />
+                </h4>
+                <h5 className="checkoutcom-sepa-title">{ `${debtor?.firstName} ${debtor?.lastName}` }</h5>
+                <p className="checkoutcom-sepa-line">{ debtor?.address1 }</p>
+                <p className="checkoutcom-sepa-line">{ `${debtor?.postalCode} ${debtor?.city}, ${debtor?.stateOrProvinceCode}` }</p>
+                <p className="checkoutcom-sepa-line">{ debtor?.countryCode }</p>
```

Comment:
Why is `debtor` optional? Should we null check for the `debtor` variable to avoid printing empty spaces and commas? 🤔